### DISCUSSION TITLE
Inheritance support ?

### DISCUSSION THREAD

**Discussion by baudee** [serverpod/serverpod#2092] [Ideas]
Is there any inheritance support planned in the future ? If yes, any idea when it could be out ? Could be really nice to write more generic code.

**Comment by Isakdl** [comment]
Yes this is planned :) 

See RFC: https://github.com/serverpod/serverpod/issues/1873

Related issues: 
https://github.com/serverpod/serverpod/issues/1618
https://github.com/serverpod/serverpod/issues/793
